docs/src/app/components/pages/discover-more/Showcase.js

```diff
@@ -28,6 +28,12 @@ const appList = [
   //   link: '',
   // },
   {
+    title: 'SlimChess - Play Instant Chess Online Games on the Go',
```

Comment:
That's something to consider, I like the idea. But that would be for another PR. I think that we should wait for the next branch to be released before going forward in a showcase reorganization.